### ISSUE TITLE
Support Node 0.12

### ISSUE THREAD

**Issue by kitsonk** [theintern/intern#372]
It appears currently Intern has issues with running remote tests under NodeJS 0.12.  Currently I get several of the following errors when attempt to run under NodeJS 0.12:

```
Listening on 0.0.0.0:9000
Starting tunnel...
Ready
Using no proxy for connecting to Sauce Labs REST API.
Error: [POST http://(redacted)@localhost:4444/wd/hub/session] connect ECONNREFUSED
Error: connect ECONNREFUSED
  at exports._errnoException  <util.js:746:11>
  at TCPConnectWrap.afterConnect [as oncomplete]  <net.js:983:19>
```

I am not sure exactly where this is actually occurring to isolate further (or what has changed in Node to cause it).

**Comment by jason0x43** [comment]
This was an issue with the SauceLabsTunnel in Dig Dug. The tunnel code was watching for a change event on a "ready" file; this file is touched by the Sauce Connect launcher when the tunnel is ready. In previous versions of Node, the first event received by the file watcher happened when the ready file was touched. In Node 0.12, a change event is being emitted before sc touches the file, which caused Dig Dug to pre-maturely report that the sc tunnel was ready.

Dig Dug master now verifies that the ready file was actually modified when a change event is received.

**Comment by vasikarla** [comment]
is this issue resolved?

**Comment by jesucarr** [comment]
No it's not resolved, I just run into it. I had to downgrade to node v0.10 to be able to run the test. It looks like this fix has been added to milestone 3.0, although should be a fix to the current version too...
